.prettierrc

```diff
@@ -0,0 +1,7 @@
+{
+  "printWidth": 80,
+  "tabWidth": 2,
+  "semi": false,
```

Comment:
Having semicolon in the end of each statements will help our eyes to catch them better.

Reply:
I disagree. semicolons are symbols, symbols decrease the readability of code. If you look at Python, it's known for how easy it is to learn and one of the main factors is that it's near to the human language by using indentations with its few symbols. So symbols are not part of the human language. I find it more readable.